src/components/List/ApplicationsList.tsx

```diff
@@ -45,26 +45,20 @@ const ApplicationsList: React.FC<ApplicationsListProps> = ({
           </Table.Row>
         </Table.Header>
         <Table.Body>
-          {loading ? (
-            <Table.Row colSpan={columns.length} textAlign="center">
-              <Table.Cell colSpan={columns.length} textAlign="center">
-                <Loading />
-              </Table.Cell>
-            </Table.Row>
```

Comment:
`<Loading>` inside here doesn't look good as it appears in a white box. We weren't seeing this previous cos the loading prop wasn't actually coming through at all (as the whole component didn't render if Loading), but now 

We already have a `<Loading>` component below that shows up now.

Reply:
Oh good to know Loading on Applications list is fixed 👏

Reply:
Oh, although the "No applications found" warning still showing on the bottom below the Loading : (

Reply:
OH -- good catch! Will fix...

Reply:
Done 59d0e2ff :)